### ISSUE TITLE
Vending Machine Structure Generation Throws Warnings in Log When Generating in Villages

### ISSUE THREAD

**Issue by Darkosto** [Lupicus/VendingMachine#11]
Hello!

I've been having a lot of fun with the mod but I've run into a bit an issue. The Vending Machines are functional but whenever they are generated, there is a warning in the log:

```
[18:56:10] [Server thread/WARN] [minecraft/LevelChunk]: Tried to load a block entity for block Block{vm:vending_machine}[bottom=false,facing=south] but failed at location BlockPos{x=-87, y=165, z=-221}
[18:56:10] [Server thread/WARN] [minecraft/LevelChunk]: Tried to load a block entity for block Block{vm:vending_machine}[bottom=false,facing=north] but failed at location BlockPos{x=-75, y=170, z=-312}
[18:56:18] [Server thread/WARN] [minecraft/LevelChunk]: Tried to load a block entity for block Block{vm:vending_machine}[bottom=false,facing=south] but failed at location BlockPos{x=249, y=172, z=-166}
[18:56:18] [Server thread/WARN] [minecraft/LevelChunk]: Tried to load a block entity for block Block{vm:vending_machine}[bottom=false,facing=south] but failed at location BlockPos{x=252, y=170, z=-86}
```

Is there any way this warning can be looked at or suppressed? It's making it tough when running a server or world because these errors are taking up a lot of space every time a new Vending Machine is generated. 

Thank you, Have a great rest of your day!
Darkosto

**Comment by Lupicus** [comment]
This happens in newer versions (1.17+) and not in older versions (1.16.x) of forge. Forge use to be able to skip non existent block entities, but in later versions, they removed their code and opted to use Mojang's code which prints those messages only during generating world and not other times, but have no issue except the messages. I think an easy way to remove the message is to create a dummy block entity for the top block of the vending machine.

**Comment by Lupicus** [comment]
fixed in 1.0.8
